exotic/exotic.py

```diff
@@ -2340,18 +2416,28 @@ def main():
                             arrayNormUnc = arrayNormUnc * np.sqrt(myfit.chi2 / myfit.data.shape[0])  # scale errorbars by sqrt(rchi2)
                             minAnnulus = annulusR  # then set min aperature and annulus to those values
                             minAperture = apertureR
-                            # gets the centroid trace plots to ensure tracking is working
-                            finXTargCentArray = np.array(xTargCent)
-                            finYTargCentArray = np.array(yTargCent)
-                            finXRefCentArray = np.array(xRefCent)
-                            finYRefCentArray = np.array(yRefCent)
+
+                            if len(xTargCent) > 0:
+                                # gets the centroid trace plots to ensure tracking is working
```

Comment:
how does this work given that Tamim aligns all of the images up?

Reply:
This happens after the images are aligned. The lines below this save the  centroid positions from the PSF fits so they can get plotted later.